server_main.go

```diff
@@ -16,6 +16,7 @@ func main() {
 	senderAddress := flag.String("sender_address", ":6543", "The bind address for the tty-share TLS connections. tty-share tool will connect to this address.")
 	url := flag.String("url", "http://localhost", "The public web URL the server will be accessible at. This will be sent back to the tty-share tool to display it to the user.")
 	frontendPath := flag.String("frontend_path", "", "The path to the frontend resources. By default, these resources are included in the server binary, so you only need this path if you don't want to use the bundled ones.")
+	scrollBack := flag.Int("scroll_back", 1024, "Set scrollback buffer size for viewers")
```

Comment:
I would like to keep this away from the server side, and have it instead a config on the client side. I see it more of an option that the user can set in the UI/frontend. Perhaps together with other settings that change other things too (e.g.: cursor blinking, font size, etc).